internal/web/server.go

```diff
@@ -429,6 +431,7 @@ func (s *server) packages(w http.ResponseWriter, r *http.Request) {
 func (s *server) packageDetail(w http.ResponseWriter, r *http.Request) {
 	pkgName := mux.Vars(r)["pkgName"]
 	pkg, status, manifest, latestVersion, err := describe.DescribePackage(r.Context(), pkgName)
+	autoUpdate := AutoUpdateString(pkg, "Disabled")
```

Comment:
This part can stay of course, since you want to show the auto update value on the package detail page.